docs/compounds/nav-category.html

```diff
@@ -6,49 +6,48 @@
   <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
   <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="/cocoandeve-styleguides/js/main.js"></script>
   <script src="/cocoandeve-styleguides/js/nav-category.js"></script>
 </head>
 
-<body>
+<body style="height: 150vh;">
   <div class="container">
-    <div class="row">
-      <div class="col text-center mb-2 mt-2 nav-category">
-        <ul class="nav justify-content-center align-items-center">
-          <li class="nav-item">
-            <a href="#" class="nav-link active pb-0 pt-0">ALL</a>
-          </li>
-          <li class="nav-item">
-            <a href="#" class="nav-link pb-0 pt-0">HAIR</a>
-          </li>
-          <li class="nav-item">
-            <a href="#" class="nav-link pb-0 pt-0">TAN</a>
-          </li>
-          <li class="nav-item">
-            <a href="#" class="nav-link pb-0 pt-0">BODY</a>
-          </li>
-          <li class="nav-item">
-            <a href="#" class="nav-link pb-0 pt-0">HOW TO'S</a>
-          </li>
-          <li class="nav-item d-lg-none">
-            <a href="#" class="nav-link pb-0 pt-0 link-search sni sni__search text-primary">
-            </a>
-          </li>
-        </ul>
-        <div class="search-box input-group input-group-sm d-lg-none d-none mt-2">
-          <div class="input-group-prepend">
-            <span class="input-group-text search-box__icon">
-              <i class="sni sni__search"></i>
-            </span>
-          </div>
-          <input type="text" class="form-control" id="exampleSearch" placeholder="Search">
-          <div class="input-group-append">
-            <a class="input-group-text search-box__close align-items-center" href="#" role="button">
-              <i class="sni sni__close"></i>
-            </a>
-          </div>
-          <div class="input-group-append">
-            <button class="search-box-btn btn btn-primary" type="submit">Search</button>
-          </div>
+    <div class="text-center mb-2 mt-2 nav-category">
```

Comment:
this can be a unique tag with all the classes. Please also use appropriate HTML tags, in this case NAV is better.